README.md

```diff
@@ -1,93 +1,126 @@
 # Zingo Android and iOS apps
-
 App Store: [https://apps.apple.com/app/zingo/id1668209531](https://apps.apple.com/app/zingo/id1668209531)  
 Google Play: [https://play.google.com/store/apps/details?id=org.ZingoLabs.Zingo](https://play.google.com/store/apps/details?id=org.ZingoLabs.Zingo)
 
-## iOS build instructions
-
-### Prerequisites
-1. `yarn`
-2. `nodejs` recommended version v17 or higher
-3. Install Rust
-4. Add the ios targets `rustup target add aarch64-apple-ios x86_64-apple-ios`
-5. `cargo install cargo-lipo`
-6. `cargo install cbindgen`
-7. `sudo gem install cocoapods` to install cocoapods
-
-### Building
-1. In the `./rust/ios` directory, run `./build.sh`.
-   This step will take a long time.
-2. In the `./ios` directory, run `pod install`
-3. From the root `./` of the project, run `yarn install`
-4. Run `yarn react-native start` to start the dev server
-5. Run `yarn run ios` to install the app on an emulator/connected device.
+# iOS
+
+## Prerequisites
+1. Yarn
+2. NodeJS (recommended version 17+)
+3. Rust (https://www.rust-lang.org/tools/install)
+4. Rustup iOS targets (`rustup target add aarch64-apple-ios x86_64-apple-ios`)
+5. Cargo-lipo (`cargo install cargo-lipo`)
+6. Cbindgen (`cargo install cbindgen`)
+7. Cocaopods (`sudo gem install cocoapods`)
+
+## Building
+1. In the `./rust/ios` directory, run:
+      `./build.sh`.
+   This step may take a long time.
+2. In the `./ios` directory, run:
+      `pod install`
+3. From the root of the project, run:
+      `yarn install`
```

Comment:
In my experience the logical order of this steps is:
1. `yarn install` (you can omit `install` if you want to), just `yarn`
2. `pod install`
3. `./build.sh`
IMO.